src/de/espend/idea/laravel/controller/ControllerCollector.java

```diff
@@ -27,6 +27,12 @@ public static void visitControllerActions(@NotNull final Project project, @NotNu
             addAll(PhpIndex.getInstance(project).getAllSubclasses("\\App\\Http\\Controllers\\Controller"));
         }};
 
+        HashSet<String> commonTraits = new HashSet<String>();
```

Comment:
better use more abstract type. `Set<String> commonTraits =`. Also we should use a `static` class variable, its used often but collection will never change on runtime